test/e2e/utils/utils.go

```diff
@@ -195,10 +196,69 @@ func GetBlockSizeInGb(instance *remote.InstanceInfo, devicePath string) (int64,
 	return utilcommon.BytesToGb(n), nil
 }
 
+func Symlink(instance *remote.InstanceInfo, src, dest string) error {
+	output, err := instance.SSH("ln", "-s", src, dest)
+	if err != nil {
+		return fmt.Errorf("failed to symlink from %s to %s. Output: %v, errror: %v", src, dest, output, err)
+	}
+	return nil
+}
+
 func RmAll(instance *remote.InstanceInfo, filePath string) error {
 	output, err := instance.SSH("rm", "-rf", filePath)
 	if err != nil {
 		return fmt.Errorf("failed to delete all %s. Output: %v, errror: %v", filePath, output, err)
 	}
 	return nil
 }
+
+func MkdirAll(instance *remote.InstanceInfo, dir string) error {
+	output, err := instance.SSH("mkdir", "-p", dir)
+	if err != nil {
+		return fmt.Errorf("failed to mkdir -p %s. Output: %v, errror: %v", dir, output, err)
+	}
+	return nil
+}
+
+func CopyFile(instance *remote.InstanceInfo, src, dest string) error {
+	output, err := instance.SSH("cp", src, dest)
+	if err != nil {
+		return fmt.Errorf("failed to copy %s to %s. Output: %v, errror: %v", src, dest, output, err)
+	}
+	return nil
+}
+
+// ValidateLogicalLinkIsDisk takes a symlink location at "link" and finds the
+// link location - it then finds the backing PD using scsi_id and validates that
```

Comment:
This is fine for now, although when NVMe comes up, this test will get outdated. I think writing data and verifying it would be more portable.

Reply:
The thing is writing data would work if the symlink was made incorrectly, so I'm not sure it validates what we care about. (Which is that the volume matches the device). Writing this check was much harder than writing data :)

I don't think network attached PD supports NVME. Only available for local ssd

Reply:
Yes, there's no NVME for PD now, I'm just saying for future proofing.